.github/workflows/pr.yaml

```diff
@@ -1,32 +1,79 @@
-name: Develop PR - build, test
+name: Develop PR - build test
 
 on:
   pull_request:
     branches:
       - develop
 
 jobs:
-  build:
-    strategy:
-      matrix:
-        include:
-          - component: Frontend
-            location: ui
-          - component: Backend
-            location: .
+  backend:
     runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          path: ./
+
+      -
+        name: Install just
+        uses: extractions/setup-just@v1
+
+      -
+        name: Install Go
+        uses: actions/setup-go@v4
+        with:
+          go-version: '1.19'
```

Comment:
I believe we can use `stable` here as long as there aren't version dependencies in the build.

Reply:
Our `go.mod` does specify a Go version, but not because of any hard dependency. I'm definitely supportive of using latest stable here. I've updated this PR accordingly (and the KCM PR the same).

> I am interested in what we have been doing here to date

We've been using latest stable AFAIK -- all of our Dockerfiles are `golang:latest`.